nix/packages/default.nix

```diff
@@ -1,259 +1,267 @@
-{inputs, ...}: {
-  perSystem = {
-    system,
-    pkgs,
-    unstablePkgs,
-    ...
-  }: let
-    inherit (pkgs) stdenv;
-
-    src = ../../.;
-    root = src;
-  in {
-    packages = rec {
-      upstream-nim-codetracer = pkgs.buildPackages.nim1.overrideAttrs (old: {
-        postInstallPhase = ''
-          mv $out/nim $out/upstream-nim
-        '';
-      });
-
-      noir = inputs.noir.packages.${system}.default;
-
-      sqlite = pkgs.sqlite;
-
-      pcre = pkgs.pcre;
-
-      chromedriver-102 = pkgs.chromedriver.overrideAttrs (old: {
-        version = "102.0.5005.27";
-        src = builtins.fetchurl {
-          url = "https://chromedriver.storage.googleapis.com/102.0.5005.27/chromedriver_linux64.zip";
-          sha256 = "sha256:1978xwj9kf8nihgakmnzgibizq6wp74qp2d2fxgrsgggjy1clmbv";
-        };
+{ inputs, ... }:
```

Comment:
why is default.nix reordered: autoformatting? has something changed in the logic of it?

Reply:
i assume so: merging